charts/filtered-resource-service/templates/ingress-route.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018-2021 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: {{ include "filtered-resource-service.name" . }}
+  namespace: {{ include "palisade.namespace" . }}
+spec:
+  entryPoints:
+    - web
+  routes:
+    - match: Host(`{{ .Values.global.hostname }}`) && PathPrefix(`{{ include "filtered-resource-service.root" . }}/filteredResource`)
```

Comment:
using the service name in mixed case as the route url is a terrible idea and kind of exposes function, I recommend making this just /resource

Reply:
I can see that the problem is that I missed something when the service was developed. The path on the service should be just / so that routing becomes /resource/xxx-xxx-xxx-xxx, so make it resource/resource for now and raise a tiny ticket to correct the service.

Reply:
The issue here is the service has other endpoints (`/actuator/health`) so you'd have to not confuse the two:
`/actuator` the endpoint for health and stuff
vs
`/actuator` because for some reason the token for the request was "actuator"
I'd change the `/resource` path in the service to be more like `/fetchResources` or similar